Modification from Nathalie and Romain's feedbacks. (#413)
akeneo/pim-api-docs: content/guides/translation-connection/step3-how-to-build-your-connector.md

After this last action, the bulk action will run and Julia will be notified in the Akeneo PIM notification bar once it is successfully executed.

With these new attributes, your connector is now ready to go! Let's see how it'll behave on the other side of the connector.

### How it works with your connector?

> **Find products with the enabled "translation queued" status**

Reviewer: ```suggestion
**Find products with the "translation queued" status enabled**
```

At regular intervals, your connector will need to retrieve products where the `Translation queued` attribute has been set to enable (set to `true`).

You can do this by using our PIM [API filtering system](https://api.akeneo.com/documentation/filter.html#filter-on-product-values) on product attribute value.

**Tell Julia that her translation project is "in progress"**

> Then, your connector needs to set this attribute to `false` and change the `Translation status` attribute to `IN PROGRESS` to indicate Julia that the translation is being processed by one of your translators.

Reviewer 2: Why do you need to set the attribute "translation queued" to false? It could be good to precise why? I guess you are doing that in case an updated translation was needed?

Author: Sure, "translation queued" is to set that this product needs to be translated for the connector. So after this connector has retrieved the product, it is necessary to change this state in the PIM to prevent it from being taken into consideration when the connector re-processes PIM products.

Author: I add a comment in my documentation for that.

**Retrieve "text" product attributes**

You have 2 possibilities to develop this action:

> * The simplest but less automatic way is that, in your connector configuration, Julia and Peter have previously declared the exhaustive list of text attributes to be translated (by product family). Then, your connector has to consult this list according to the family.

Reviewer: ```suggestion
* The simplest but less automatic way is that, directly in your connector configuration, Julia and Peter declare the exhaustive list of text attributes to be translated (by product family). Then, your connector has to consult this list one family at a time.
```

> * The other way is that your connector can analyze the PIM product families to determine which attributes have a "text" type and can be translated (check that the attribute is "localizable" too).

Reviewer: ```suggestion
* The other way is that your connector can analyze the PIM product families to determine which attributes have a "text" type and, therefore, can be translated. It must also check that the attribute is "localizable".
```

:::info

> We suggest that you develop the first solution during a first iteration of your connector and then, if you encounter Julias with a large number of product families with many attributes to translate, you can then upgrade your connector with the second solution.

Reviewer: ```suggestion
We suggest that you develop the first solution during the first iteration of your connector. If in your projects Julia handles a large number of product families with many attributes to translate, you can then upgrade your connector with the second solution.
```

By analyzing these types, you will be able to define for your translator which are the textual attributes they need to translate.
:::

> **What about simple and multi select attribute type?**

Reviewer: ```suggestion
**What about simple and multi select attribute types?**
```

> We mentioned [earlier](step2-understand-akeneo-pim.html#pim-product-data-that-can-be-translated) that it would be interesting for your connector to offer translation of "simple" or "multi select" type attributes.

Reviewer: ```suggestion
We mentioned [earlier](step2-understand-akeneo-pim.html#pim-product-data-that-can-be-translated) that it would be interesting for your connector to allow the translation of "simple" or "multi select" type attributes.
```

> To do this, Julia and Peter can declare the exhaustive list of "simple" or "multi select" attributes in your connector configuration, just as they do for "text" attributes.

Reviewer: ```suggestion
To do so, Julia and Peter can declare the exhaustive list of "simple" or "multi select" attributes in your connector configuration, just as they do for "text" attributes.
```

> When Julia requests the translation of a product, your connector can then take the opportunity to translate the "options" of these "simple" or "multi select" attributes that have not yet been translated.  

Reviewer: ```suggestion
When Julia requests the translation of a product, your connector will then automatically translate the "options" of the "simple" or "multi select" attributes that have not been translated yet.  
```

:::tips
To perform this action, your connector can use the ["get list of attribute options"](https://api.akeneo.com/api-reference.html#get_attributes__attribute_code__options) API endpoint.
:::

> **Tell Julia that her translation project is finished...**

Reviewer: ```suggestion
**Tell Julia that her translation project is either finished...**
```
